skimage/transform/_warps.py

```diff
@@ -831,8 +834,9 @@ def warp(image, inverse_map, map_args={}, output_shape=None, order=1,
             matrix = inverse_map.params
 
         elif (hasattr(inverse_map, '__name__') and
-              inverse_map.__name__ == 'inverse' and
+              inverse_map.__name__ == 'inverse_map' and
               get_bound_method_class(inverse_map) in HOMOGRAPHY_TRANSFORMS):
+            # TODO: do we still need this introspection hack?
```

Comment:
This case is no longer needed since both `inverse` and `inverse_map` are properties that return `GeometricTransform` and `np.ndarray` respectively, which are covered by the two previous cases.

Reply:
@hmaarrfk Disregard what I said about removing this case; I forgot that `inverse_map` is actually a function so this still applies.

However, to answer your question, you shouldn't need this introspection hack any more. I'll explain in additional comments.